pod_test.go

```diff
@@ -280,8 +280,8 @@ func TestPodCollector(t *testing.T) {
 				},
 			},
 			want: metadata + `
-				kube_pod_info{host_ip="1.1.1.1",namespace="ns1",pod="pod1",node="node1",pod_ip="1.2.3.4"} 1
-				kube_pod_info{host_ip="1.1.1.1",namespace="ns2",pod="pod2",node="node2",pod_ip="2.3.4.5"} 1
+				kube_pod_info{controllers="<none>",host_ip="1.1.1.1",namespace="ns1",pod="pod1",node="node1",pod_ip="1.2.3.4"} 1
```

Comment:
Why do we name it controller to begin with? The annotation says `created_by` and not `controller`. Just for people to intuitively understand what it was populated by, I'd avoid introducing name mappings.

Anyone knows whether that annotation is scheduled to move into the main YAML and if, whether it will keep the name?